src/components/Members Section/index.tsx

```diff
@@ -0,0 +1,2 @@
+export * from './MembersSection';
+export { default } from './MembersSection';
```

Comment:
why do we need to do it like this, can we not rename the MembersSection.tsx file to index.js as it is part of MembesSection folder only and default export the component from there only

Reply:
I think I did it after going through this article which was linked to in the repo's readme which shows an example for the folder structure.
https://www.joshwcomeau.com/react/file-structure/

Reply:
Hey @vinayak-trivedi seems like a fair point from Nabajit as adding `index.js` in your folder for `MembersSection ` Component will thus populate our codebase with too many index.js files.  Although this implementation seems reasonable to me will still want to know if you have any specific concerns about this approach

Reply:
@NabajitS please rename your folder with ~`Members Section`~ ->  `MembersSection`

Reply:
Renaming the folder shows its inner files as deleted and created again. IG this is normal
![Screenshot 2023-04-02 210454](https://user-images.githubusercontent.com/86390740/229364762-4e47faee-f898-4496-9a0a-f87e8441dbab.png)